src/tcgwars/logic/impl/gen3/HolonPhantoms.groovy

```diff
@@ -2173,15 +2242,17 @@ public enum HolonPhantoms implements LogicCardInfo {
         text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card." +
           "Discard a card from your hand. If you can't discard a card from your hand, you can't play this card. Draw 3 cards. If you discarded a Pokémon that has δ on its card, draw 4 cards instead."
         onPlay {
-        my.hand.getExcludedList(thisCard).select("Choose a card to discard.").discard().each {
+          my.hand.getExcludedList(thisCard).select("Discard a card from your hand in order to play ${thisCard}.").discard().each {
             draw 3
             if(it.cardTypes.is(POKEMON) && it.cardTypes.is(DELTA)){
               draw 1
             }
           }
         }
         playRequirement{
-          assert my.hand.getExcludedList(thisCard)
+          def hand = my.hand.getExcludedList(thisCard).size() >= 1
+          assert hand : "One other card in hand is required to play this card."
```

Comment:
A size greater than 0 is already implied by an empty list being falsy. This variable definition doesn't really accomplish anything as far as I can tell.
```suggestion
          assert my.hand.getExcludedList(thisCard) : "One other card in hand is required to play this card."
```

Reply:
Yup, sounds good. Focused on consistency and should've fixed it already on last set. Will apply this to all holon supporters 🙂